operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -7,16 +7,28 @@ include::modules/common-attributes.adoc[]
-Scanner analyzes all image layers to check for known vulnerabilities by comparing them with the Common Vulnerabilities and Exposures (CVEs) list.
+With {product-title} you can analyze images for vulnerabilities using the default {product-title-short} scanner, or you can xref:../integration/integrate-with-image-vulnerability-scanners.adoc#integrate-with-image-vulnerability-scanners[configure an integration] to use another supported scanner. The  scanners in {product-title-short} analyze all image layers to check for known Common Vulnerabilities and Exposures (CVEs) by comparing them with a vulnerability database populated from different sources. Depending on the scanner used, sources include the National Vulnerability Database (NVD), the Open Source Vulnerabilities (OSV) vulnerability database, and operating system OVAL feeds.
 
-When Scanner finds any vulnerabilities, it:
+[NOTE]
+====
+The {product-title-short} Scanner V4 uses the OSV database available at link:https://osv.dev/[OSV.dev] under license.
+====
 
-* Shows them in the xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management[*Vulnerability Management*] view for detailed analysis.
-* Ranks vulnerabilities according to risk and highlights them in the RHACS portal for risk assessment.
-* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies].
+{product-title-short} contains two scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner was the default scanner in {product-title-short} before Scanner V4 was introduced in release 4.4. The StackRox Scanner originates from a fork of the Clair v2 open source scanner.
+[NOTE]
+====
+This documentation uses the term "{product-title-short} scanner" or "Scanner" to refer to the combined scanning capabilities provided by the two scanners: the StackRox Scanner and Scanner V4. When referring to the capabilities of a specific scanner, the name of the specific scanner is used.
+====
 
-Scanner inspects the images and identifies the installed components based on the files in the images.
+When the {product-title-short} scanner finds any vulnerabilities, it performs the following actions:
+
+* Shows them in the xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management[*Vulnerability Management*] view for detailed analysis
+* Ranks vulnerabilities according to risk and highlights them in the {product-title-short} portal for risk assessment
+* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies]
+
+The {product-title-short} scanner inspects the images and identifies the installed components based on the files in the images.
+//verify the following guidance still applies
+#Does this still apply to Scanner V4 also?#
 It may fail to identify installed components or vulnerabilities if the final images are modified to remove the following files:
```

Comment:
yes, see https://docs.google.com/document/d/1A62fyY-MZMZq3mfBY5ygUFxOhdAmiAOI_VyR4QMlO-4/edit#heading=h.hibnbrdbkisz and https://docs.google.com/document/d/1A62fyY-MZMZq3mfBY5ygUFxOhdAmiAOI_VyR4QMlO-4/edit#heading=h.iexadqqzmvu0